src/zupply.cpp

```diff
@@ -12480,19 +12493,72 @@ namespace zz
 
 			// type info
 			std::string st = "<" + type_ + ">";
+			std::vector<std::string> stvec;
+			if(type_.find('\n') != std::string::npos)
```

Comment:
I don't understand what this block for?

Reply:
Due to the original "quick" solition that came to mind and the personal preference of not disturbing your code too much, I decided to use a separator character in the `type_` string (which was `\n`, as explained in a previous comment) to indicate each element in the list of possible types.

This block basically checks if a newline was used or not. If it's used, create a list of all the different types, otherwise, act normal. If the list is incomplete, fill with empty types until `max_`.
An empty list will change nothing to the current inner workings of the code, while it otherwise would display the types according to their occurence in the list. 
e.g. If the list contains `type1`, `type2` and `type3`; the option `-c` that uses it would show `-c <type1> <type2> <type3>` instead of `-c <type1 type2 type3> <> <>`